### ISSUE TITLE
Feature Request: Social Groupings / Communities

### ISSUE THREAD

**Issue by rodrigoalonso-carb** [monicahq/monica#7319]
### How to use GitHub

- Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
- Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
- Subscribe to receive notifications on status change and new comments.

**Is your feature request related to a problem? Please describe.**
- Monica doesn’t have a feature to add certain contacts to “Social Groups” or “Communities”. I would like to click on “Friday Night TableTop Game Group” and see the list of people that are part of that Group. 

**Describe the solution you'd like**
- As a person with defined social groups, I would like the ability to add contacts to groups/communities (multiple). 
- I could then click on “Friday Night TableTop Games” groups and see a list of the people in said group. I could click on each person to go to their contact page so I can see past notes. 
- A bonus would be to add notes for the groups, and reminders also to the group. Treated as a Contact, but with a list of contacts. 

**Describe alternatives you've considered (optional)**
The alternative is to remember who are the regulars of my different communities to invite them and review the notes beforehand.  however there are people that I forget to invite and people that go without invitation that I didn’t read my notes on. 

**Additional context**
I pertain to several different communities with regular meetings and in each meeting I always forget to invite x person, or even forget their names. Any simple solution or other ideas are we

Thank you for considering my enhancement, and thank you so much for this life changing product, I am very grateful!

**Comment by jawadakhan97** [comment]
Hi, there is a 'group' setting though creating one is a bit hidden.

If you go to: Settings>Personalize Your Account>Manage Group Types then you can define or create groups. I made one called 'School' and added 'Roles' for 'College' and 'High School'.

Then if you go to a contact, Contact>Social>Add to Group you can add the contact to a group or role and I think that could help for now.

 It would be nice to have a way to create a mailing list from a group or role in a group. I haven't seen anything like that, though I might have missed it!

**Comment by rodrigoalonso-carb** [comment]
Hi, thanks for the response, I could not find the setting you mention, however I found a tagging system which is good enough for groups. It does not add 'Roles' (I loved your example of 'regular' vs 'occasional') but the tags are the basic functionality I need.

Regarding the settings I couldn't find:
I am on self hosted version 4.1.2

I see Settings > Account Settings which has 'International Settings' and 'Layout'
and Setting > Personalization which has 'Gender Type', 'Field Type', 'Activity Type', 'Life Event' and 'Features'

I also couldn't find Contact > Social anywhere or >Add to Group under any setting in my contact page. 

Thanks!

**Comment by rodrigoalonso-carb** [comment]
The way I obtained the functionality I needed was to add tags and add each person in the group to the tag. I then created a Contact with the name of the Tag + Group. 

So I have "Friday Table Games Group" as a contact with the "friday table games" tag and "Roomies D&D Group" as another contact with the "roomies dnd" tag. 

In these group contacts is where I add the notes to each social group that I need to remember. When a person joins the group I add the respective tag to the person contact, and voila, I have everything I need.
